Updated hooks, added new features

Unbrick opened the pull request:
Updated hooks to 4.11.2
Added color changing mod
Seamless integration into Jodel App in the ListView of the Right View pager
Updated build version to 24
Removed unneccesary / unused classes and resources

Unbrick committed:
Updated hooks to 4.10.3 (1000005) .

Unbrick committed:
Updated hooks to 4.10.4 (1000006) .

Unbrick committed:
Updated hooks to 4.10.4 (1000006) .

Unbrick committed:
- Updated hooks to 4.11.2 (1000009)
 - Added color change mod for posts
 - Implemented JodelXposed seamless in JodelApp (Swipe Right)
 - Removed SettingsActivity and its resources, not in use anymore

krokofant commented:
Did you forget to *git add* the *BackgroundOperations* file? 😮 Hyped to test it out.

Unbrick committed:
- Updated hooks to 4.11.2 (1000009)
 - Added color change mod for posts
 - Implemented JodelXposed seamless in JodelApp (Swipe Right)
 - Removed SettingsActivity and its resources, not in use anymore
 - Added BackgroundOperations.class (damn default settings)

Unbrick commented:
Yes, Android Studio used its default settings, fixed and Pushed!
